docker/pypi/wmagent/run.sh

```diff
@@ -1,5 +1,29 @@
 #!/bin/bash
 
+_find_child_pss() {
+    # An auxiliary function to find all processes forked from the shell parrent
+    # of the current process
+    cat  /proc/$PPID/task/*/children
+}
+
+_service_gracefull_exit() {
+    # An auxiliary function to handle the WMAgent service graceful exit
+    manage stop-agent
+    wait
```

Comment:
This seems to wait for all of the "background processes" to finish, before it can move on. 
We might have to add some debugging when testing these changes to verify where components are stopped (gracefully vs -9).

Reply:
this  was  removed in the latest  commit... for other reasons